src/client/src/components/Chat.jsx

```diff
@@ -0,0 +1,75 @@
+import { TextField } from "@mui/material";
+import React, { useContext, useEffect } from "react";
+import "../css/Chat.css";
+import styled from "@mui/material";
+import SendRoundedIcon from "@mui/icons-material/SendRounded";
+import AttachFileIcon from "@mui/icons-material/AttachFile";
+
+const Chat = () => {
+  return (
+    <div className="conatiner">
+      <div className="chatbox">
+        <div className="userpic">
+          <h2 className="username">Jane Doe</h2>
+        </div>
+        <div class="vl"></div>
+        <div className="sender-message">
+          <div className="sender-userpic"></div>
+          <div className="sender-text">Hi, how are you?</div>
+        </div>
+        <div className="user-message">
+          <div className="user-text">Hi, I am doing great! What about you?</div>
+          <div className="user-userpic"></div>
+        </div>
```

Comment:
This div is on top of the message input. You need to add a margin.
![Screen Shot 2023-03-09 at 11 17 09 AM](https://user-images.githubusercontent.com/14958097/224086455-ce6e5ea2-eaec-46bf-91af-59138c3bfd76.png)